functions/Remove-DbaClientAlias.ps1

```diff
@@ -1,123 +1,110 @@
+            Allows you to login to remote computers using alternative credentials
 
-    .PARAMETER Alias
-    The alias to be deleted
+        .PARAMETER Alias
+            The alias or array of aliases to be deleted
 
-    .PARAMETER EnableException
-    By default, when something goes wrong we try to catch it, interpret it and give you a friendly warning message.
-    This avoids overwhelming you with "sea of red" exceptions, but is inconvenient because it basically disables advanced scripting.
-    Using this switch turns this "nice by default" feature off and enables you to catch exceptions with your own try/catch.
+        .PARAMETER WhatIf
+            If this switch is enabled, no actions are performed but informational messages will be displayed that explain what would happen if the command were to run.
 
-    .NOTES
-    Tags: Alias
+        .PARAMETER Confirm
+            If this switch is enabled, you will be prompted for confirmation before executing any operations that change state.
 
-    Website: https://dbatools.io
-    Copyright: (C) Chrissy LeMaire, clemaire@gmail.com
-    License: MIT https://opensource.org/licenses/MIT
+        .PARAMETER EnableException
+            By default, when something goes wrong we try to catch it, interpret it and give you a friendly warning message.
+            This avoids overwhelming you with "sea of red" exceptions, but is inconvenient because it basically disables advanced scripting.
+            Using this switch turns this "nice by default" feature off and enables you to catch exceptions with your own try/catch.
 
-    .LINK
-    https://dbatools.io/Remove-DbaClientAlias
+        .NOTES
+            Tags: Alias
 
-    .EXAMPLE
-    Remove-DbaClientAlias -ComputerName workstationx -Alias sqlps
-    Removes the sqlps SQL client alias on workstationx
+            Website: https://dbatools.io
+            Copyright: (C) Chrissy LeMaire, clemaire@gmail.com
+            License: MIT https://opensource.org/licenses/MIT
 
-    .EXAMPLE
-    Get-DbaClientAlias | Remove-DbaClientAlias
-    Removes all SQL Server client aliases on the local computer
+        .LINK
+            https://dbatools.io/Remove-DbaClientAlias
 
-#>
-    [CmdletBinding()]
-    Param (
-        [parameter(ValueFromPipelineByPropertyName)]
+        .EXAMPLE
+            Remove-DbaClientAlias -ComputerName workstationx -Alias sqlps
+
+            Removes the sqlps SQL client alias on workstationx
+
+        .EXAMPLE
+            Get-DbaClientAlias | Remove-DbaClientAlias
+
+            Removes all SQL Server client aliases on the local computer
+    #>
+    [CmdletBinding(SupportsShouldProcess = $true)]
+    param (
+        [parameter(ValueFromPipelineByPropertyName = $true)]
         [DbaInstanceParameter[]]$ComputerName = $env:COMPUTERNAME,
         [PSCredential]$Credential,
-        [parameter(Mandatory, ValueFromPipelineByPropertyName)]
+        [parameter(Mandatory = $true, ValueFromPipelineByPropertyName = $true)]
```

Comment:
Thanks so much for the update! I'll be merging this PR, but please note that `= $true` is 2.0 syntax. This will be removed en masse prior to 1.0

Reply:
I'm sorry, I'm not following. Can you elaborate? 

While I was in there I figured I would reformat it to follow the formatting from the Get-DbaSmoObject template (https://github.com/sqlcollaborative/dbatools-templates/blob/master/Get-DbaSmoObject.ps1).

Reply:
D'oh! We'll need to update that. `Mandatory = $true`  is PS 2.0 syntax and the `= $true` is no longer needed.

Reply:
Got it - good to know. Thank you!